source/deployment/services/loadbalancer.rst

```diff
@@ -265,10 +265,16 @@ Add both network id and security group id to the configuration repository.
 
 .. code-block:: yaml
    :caption: /opt/configuration/environments/kolla/configuration.yml
+
+   octavia_service_auth_project: "octavia"
+   octavia_loadbalancer_topology: "ACTIVE_STANDBY"
+   octavia_network_type: "tenant"
+   octavia_auto_configure: false
```

Comment:
Why false?

Reply:
The auto-configuration currently only support network_type `provider`. It is also using the `service` project.